modules/ROOT/pages/platforms.adoc

```diff
@@ -20,6 +20,7 @@ The currently supported platforms and their identifiers are listed below.
 * Bare metal (`metal`): With BIOS, UEFI or network boot, with standard or 4k Native disks. See xref:bare-metal.adoc[Installing on Bare Metal] or xref:live-booting-ipxe.adoc[Live-booting via iPXE].
 * Nutanix (`nutanix`): Hypervisor.
 * OpenStack (`openstack`): Cloud platform. See xref:provisioning-openstack.adoc[Booting on OpenStack].
+* Proxmox (`proxmoxve`): Hypervisor. See xref:provisioning-proxmoxve.adoc[Booting on Proxmox]
```

Comment:
Ahh nice. I think there are a few more platforms that we've added recently that also need to be updated here.